editor/effects.js

```diff
@@ -273,4 +284,106 @@ export default {
 
 		return effects;
 	},
+	// TODO: FETCH_REUSABLE_BLOCKS and FETCH_REUSABLE_BLOCK can probably be combined.
+	FETCH_REUSABLE_BLOCKS( action, store ) {
```

Comment:
can't we replace all those custom fetching actions with the `withApiData`? Maybe not because we want these in the Redux state? in which case, maybe there's something we can do here. A wrapper to `withApiData` dealing with the state or something? @aduth any thoughts on this?

Reply:
Like you say, we'd need to change `withApiData` to store the models it fetches in Redux. It could be a cool abstraction. I feel it's best to punt on it for now, though.

Reply:
This is exactly what I'm looking for today :) It would be nice to be able to have an option to use the same API without a component, e.g. inside Redux middleware.